jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Jcmd.java

```diff
@@ -40,17 +40,24 @@
 public class Jcmd {
 
 	@SuppressWarnings("nls")
-	private static final String HELPTEXT = "jcmd: run diagnostic command on another Java process%n"
-			+ " Format:%n"
-			+ "    jcmd <vmid> <arguments>%n"
-			+ " Arguments:%n"
-			+ "    <vmid>: Attach API VM ID as shown in jps or other Attach API-based tools.%n"
-			+ "    command[command arguments]: command from the list returned by \"-help\"%n"
-			+ "    -help: print the list of diagnostic commands%n"
-			+ "    -J: supply arguments to the Java VM running jcmd%n"
-			+ " NOTE: this utility may significantly affect the performance of the target JVM.%n"
-			+ "     The available diagnostic commands is determined by%n"
-			+ "     the target VM and may vary between VMs.%n";
+	private static final String HELPTEXT = "Usage : jcmd <vmid> <arguments>%n"
+			+ "%n"
+			+ "   -J : supply arguments to the Java VM running jcmd%n" 
+			+ "   -l : list JVM processes on the local machine%n"
```

Comment:
I was suggesting we indicate here this option is the default when no parameters are specificed.

Reply:
nm, I see you added this lower down.